cli/testing/fake-diff-editor.rs

```diff
@@ -84,7 +84,7 @@ fn main() {
                     exit(1)
                 }
             }
-            ["print", message] => {
+            ["file", "print", message] => {
```

Comment:
This is an argument for the fake diff-editor. It should not have to match `jj file print`.

Reply:
Done